book.rb

```diff
@@ -0,0 +1,12 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
```

Comment:
- [ ] Kindly add a method for adding rentals inside the book class to properly implement the `many to many`
between `book and person` through `rental`